components/script/dom/webidls/VRLayer.webidl

```diff
@@ -0,0 +1,13 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// https://w3c.github.io/webvr/#interface-vrlayer
+
+//typedef (HTMLCanvasElement or OffscreenCanvas) VRSource;
```

Comment:
is there any harm in introducing the `VRSource` without support in Servo for `OffscreenCanvas` yet? (we have the API already shipped in Firefox, albeit behind a flag.) filed servo issue #14627 to track the implementation of it.